### DISCUSSION TITLE
IntelliJ IDEA 2023.1 EAP create project error report

### DISCUSSION THREAD

**Discussion by code142857** [community/community#46940] [Copilot Conversations]
### Select Topic Area

Bug

### Body

```text
IntelliJ IDEA 2023.1 EAP (Ultimate Edition)
Build #IU-231.6890.12, built on February 9, 2023

com.github.copilot (1.2.3.2385)
```
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/32949827/218024257-650fcc6b-0329-4a8f-b803-8458df979448.png">

<img width="1688" alt="image" src="https://user-images.githubusercontent.com/32949827/218024219-27e666b8-02d8-4272-96c9-ab7ba3aa6278.png">

<img width="1688" alt="image" src="https://user-images.githubusercontent.com/32949827/218024498-f194bb4f-5c47-49bb-9177-33d5fc0d7219.png">

**Comment by iamcjwilliams** [comment]
@auto-generated Visit [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue
